packages/site/cms-site/CHANGELOG.md

```diff
@@ -546,7 +546,7 @@
     export const graphQLApiUrl = `${typeof window === "undefined" ? process.env.API_URL_INTERNAL : process.env.NEXT_PUBLIC_API_URL}/graphql`;
     export function createGraphQLFetch(previewData?: SitePreviewData) {
         return createGraphQLFetchLibrary(
-            createFetchWithDefaults(createFetchWithPreviewHeaders(fetch, previewData), { next: { revalidate: 15 * 60 } }),
+            createFetchWithDefaults(fetch, { next: { revalidate: 15 * 60 }, headers: { ...getGraphQLPreviewHeaders(previewData), ...headers } }),
```

Comment:
```suggestion
            createFetchWithDefaults(createFetchWithPreviewHeaders(fetch, previewData), { next: { revalidate: 15 * 60 } }),
```

I don't think we should change  changelog entries of released versions. The migration guide would be okay but not the changelog